panda/src/audio/audioSound.cxx

```diff
@@ -35,12 +35,12 @@ AudioSound() {
 
 
 void AudioSound::
-set_3d_attributes(PN_stdfloat px, PN_stdfloat py, PN_stdfloat pz, PN_stdfloat vx, PN_stdfloat vy, PN_stdfloat vz) {
+set_3d_attributes(PN_stdfloat px, PN_stdfloat py, PN_stdfloat pz, PN_stdfloat vx, PN_stdfloat vy, PN_stdfloat vz, PN_stdfloat dx, PN_stdfloat dy, PN_stdfloat dz) {
   // Intentionally blank.
 }
 
 void AudioSound::
-get_3d_attributes(PN_stdfloat *px, PN_stdfloat *py, PN_stdfloat *pz, PN_stdfloat *vx, PN_stdfloat *vy, PN_stdfloat *vz) {
+get_3d_attributes(PN_stdfloat* px, PN_stdfloat* py, PN_stdfloat* pz, PN_stdfloat* vx, PN_stdfloat* vy, PN_stdfloat* vz, PN_stdfloat* dx, PN_stdfloat* dy, PN_stdfloat* dz) {
```

Comment:
The asterisk location was correct before you changed it.  These qualifiers go around the identifier, not the type.  That's because C++ is parsed that way.

Some explanation of why it would be confusing to put it next to the type:
```cpp
int* a, b;
```
`a` is an `int*` but `b` is an `int`.  The following makes it much more obvious that b isn't a pointer:
```cpp
int *a, b;
```

That's why we put it around the identifier.